src/EditorFeatures/CSharpTest/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -1869,19 +1877,21 @@ public object GetService(Type serviceType)
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsImplementInterface)]
         public async Task TestMissingWithIncompleteMember()
         {
-            await TestMissingInRegularAndScriptAsync(
+            var code =
 @"interface ITest
 {
     void Method();
 }
 
-class Test : [|ITest|]
+class Test : ITest
 {
-    p public void Method()
+    p {|CS1585:public|} void Method()
```

Comment:
📝 In this case, the code fix is missing because CS0535 is not reported on the `ITest` identifier. If there is a different syntax which produces incomplete member errors but still produces CS0535, the results might be different than this test.